map/src/map/layers/PoiLayer.js

```diff
@@ -178,6 +195,10 @@ export default function PoiLayer() {
             {
                 params: {
                     locale: i18n.language,
+                    lat: map.getCenter().lat,
+                    lon: map.getCenter().lng,
+                    prevSearchRes: prevSearchRes,
+                    prevSearchCategory: prevSearchCategory,
```

Comment:
Possibly it's better to move all the new params into PoiSearchData class (and use vars from POST data automatically).